styles/index.css

```diff
@@ -0,0 +1,27 @@
+
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 62.5%;
```

Comment:
https://www.aleksandrhovhannisyan.com/blog/62-5-percent-font-size-trick/

Do the trick about this is put the normal "1 rem" into 10pixel instead of 16. easier for our brains to calculate when using rem for responsive mesures.